src/components/core/blocks/Checkout/OrderReview.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div id='order-review'>
+    Core order review
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'OrderReview',
+  beforeMount () {
```

Comment:
beforeMount ()  & components is empty therefore isn't required

Reply:
OK